.github/workflows/build.yml

```diff
@@ -0,0 +1,110 @@
+# This pipeline builds the sfpowerscripts plugin
+
+
+name: 'sfpowerscripts Build Pipeline'
+
+on:
+  push:
+    branches:
+      - develop
+    paths-ignore:
+      - 'docs/**'
+
+  workflow_dispatch:
+
+
+jobs:
+
+#Merge to Develop, Deploy Alpha builds
+  alpha:
+    name: 'Alpha Builds'
+    runs-on: ubuntu-latest
+    if: contains(github.ref,'refs/heads/develop')
+    environment:
+      name: alpha
+    concurrency: alpha
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: BuildCLI (sfdx-plugin)
+        uses: ./.github/workflow-templates/buildcli.yml
```

Comment:
Reusable workflows are at the job-level and can only be located within `.github/workflows`

Reply:
We can also rename the `buildcli` step & reusable workflow to something more accurate like `Build Packages`

Reply:
updated to correct usage of github reusable workflows
and
renamed the name in step for BuildCLI